request-management-api/request_api/resources/foidocument.py

```diff
@@ -73,6 +74,14 @@ def post(requestid, requesttype):
             requestjson = request.get_json() 
             documentschema = CreateDocumentSchema().load(requestjson)
             result = documentservice().createrequestdocument(requestid, documentschema, AuthHelper.getuserid(), requesttype)
+            # Add the RRT event here
+            print("THe version output is")
+            print(result.args[0])
+            ministryversion = result.args[0]
+            for document in documentschema['documents']:
```

Comment:
- remove print statements
- There is no need loop and call event class multiple times. Pass the entire schema to attachmetn event class and then check to see whether RRT is present and proceed or return.